yage-gl/README.md

```diff
@@ -0,0 +1,4 @@
+This crate should:
+* abstract over WebGL 2.0 and OpenGL 3.3+ (similar to [uni-gl](https://github.com/unrust/uni-gl), but using web_sys and ignoring WebGL 1 support)
+* provide object-oriented wrappers for GL objects (? - same crate?)
```

Comment:
I see two options. Either, make one create (this one!) for the object-oriented wrappers, and used that in the core for the actual engine objects. So we would for example have gl::Texture, which is the abstraction, and core::Texture, which is the engine object. This would make the responsibilities of both parts very clear, but may result in duplicated interfaces. Or do both in one, which would add the danger of mixing two very different layers.

Reply:
If we separate it, I wouldn't know what the difference between core::Texture and gl::Texture should/would be.
I think `yage-gl` could easily become useful apart from the rest of the engine and has to be published to crates.io anyway...something to keep in mind.